source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/README.md

```diff
@@ -0,0 +1,151 @@
+
+## SinkDataStoreProps
+
+| **Name**        | **Type**                                                                                | **Description**                                                                                                                                                                                                               |
+| :-------------- | :-------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| s3OutputBucket? | [`Bucket`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-s3.Bucket.html) | The output S3 location where the data should be written. The provided S3 bucket will be used to pass the output location to the etl script as an argument to the AWS Glue job. If not provided, the construct will create one |
+| datastoreStype  | [`SinkStoreType`](#sinkstoretype)                                                       | Sink data store type                                                                                                                                                                                                          |
+
+## SinkStoreType
+
+Enumeration of data store types that could include S3, DynamoDB, DocumentDB, RDS or Redshift. Current construct implementation only supports S3, but potential to add other output types in the future
+
+| **Name** | **Type** | **Description** |
+| :------- | :------- | --------------- |
+| S3       | `string` | S3 storage type |
+
+# Default settings
+
+Out of the box implementation of the Construct without any override will set the following defaults:
+
+### Amazon Kinesis Stream
+
+-   Configure least privilege access IAM role for Kinesis Stream
+-   Enable server-side encryption for Kinesis Stream using AWS Managed KMS Key
+-   Deploy best practices CloudWatch Alarms for the Kinesis Stream
+
+### Glue Job
+
+-   Create a Glue Security Config that configures encryption for CloudWatch, Job Bookmarks, and S3. CloudWatch and Job Bookmarks are encrypted using AWS Managed KMS Key created for AWS Glue Service. The S3 bucket is configured with SSE-S3 encryption mode
+-   Configure service role policies that allow AWS Glue to read from Kinesis Data Streams.
+
+### Glue Database
+
+-   A Glue database to add the table required to define a Glue Table structure and schema for the Kinesis stream
+
+### Glue Table
+
+-   A Table with storage descriptor and table input properties using the schema details provided for the records in the Kinesis Data Streams
```

Comment:
The description sounds too complicated and confusing, can it be simplified ?

Reply:
Ok, updating the readme